demos/_assets/css/jqm-demos.css

```diff
@@ -713,20 +720,21 @@
 	background: #ececec;
 }
 
-.jqm-navmenu-panel .ui-listview li > .ui-listview-item-button:hover,
-.jqm-navmenu-panel .ui-listview li > .ui-listview-item-button:focus,
-.jqm-navmenu-panel .ui-listview li > .ui-listview-item-button:active,
-.jqm-navmenu-panel .ui-listview li > .ui-accordion-header:hover,
-.jqm-navmenu-panel .ui-listview li > .ui-accordion-header:focus,
-.jqm-navmenu-panel .ui-listview li > .ui-accordion-header:active {
+.jqm-navmenu-panel > .ui-panel-inner > .ui-listview li > .ui-listview-item-button:hover,
+.jqm-navmenu-panel > .ui-panel-inner > .ui-listview li > .ui-listview-item-button:focus,
+.jqm-navmenu-panel > .ui-panel-inner > .ui-listview li > .ui-listview-item-button:active,
+.jqm-navmenu-panel > .ui-panel-inner > .ui-listview li > .ui-accordion-header:hover,
+.jqm-navmenu-panel > .ui-panel-inner > .ui-listview li > .ui-accordion-header:active,
```

Comment:
Right, that's because it has focus after closing. If you now tab key navigate through the menu buttons that are links get focus style but buttons that are accordion headers don't. That's bad.